pkg/roachprod/install/cluster_synced.go

```diff
@@ -1752,7 +1751,9 @@ func (c *SyncedCluster) Put(
 
 	var detail string
 	if !c.IsLocal() {
-		if c.UseTreeDist {
+		// If scp does not support the -R flag, tree dist is slower as it ends up
+		// transferring everything through this machine anyway.
+		if c.UseTreeDist && sshVersion3() {
 			detail = " (dist)"
```

Comment:
I'm confused -- this diff is changing the `detail` that is logged when transferring files, but not changing the actual logic of transferring these files. IIUC, this will change message logged on TC nightly builds (where agents  run Ubuntu 20.04) but that's confusing since the transferring algorithm won't have changed.

Am I missing something?